### ISSUE TITLE
Stuck zombies

### ISSUE THREAD

**Issue by lamefun** [Cirrus-Minor/witchblast#40]
Sometimes these kind of rooms:

![Room](http://d2.alternativeto.net/dist/s/witch-blast_614768_full.png?format=jpg&width=1600&height=1600&mode=min&upscale=false)

contain a few zombies in the room that can't move.

**Comment by Cirrus-Minor** [comment]
Yes, the problem has been precisely identified: it occurs when a monster is "stuck" at the beginning of the fight between walls and obstacles. It should begin to move when a way is free (obstacle destroyed), but it does not.
